docs/docker/graphile-migrate

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# A wrapper for running graphile-migrate within a container
+
+# Use
+# * -ti --init to allow for CTRL-C to work with `graphile-migrate watch`
+# * --user     to allow the current user to own any committed migrations
+# * mount teamapp/dbconf for access to graphile-migrate working files
```

Comment:
The terms `teamapp` and `dbconf` both only exist in this line in this PR; what do they refer to?

Reply:
Oops!  These are some names which refer to our application code. Fixed now.